LondonBritishLibrary/orient/BLorient13266.xml

```diff
@@ -0,0 +1,419 @@
+                                    <msItem xml:id="p1_i1.3.1">
+                                        <title ref="LIT2509Weddas#Sunday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.2"> 
+                                        <title ref="LIT2509Weddas#Monday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.3"> 
+                                        <title ref="LIT2509Weddas#Tuesday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.4">
+                                        <title ref="LIT2509Weddas#Wednesday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.5">
+                                        <title ref="LIT2509Weddas#Thursday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.6">
+                                        <title ref="LIT2509Weddas#Friday"/>
+                                    </msItem>
+                                    <msItem xml:id="p1_i1.3.7">
+                                        <title ref="LIT2509Weddas#Saturday"/>
+                                    </msItem>     
+                                </msItem>
+                                <msItem xml:id="p1_i1.4">
+                                    <locus from="192rb" to="195vb"/>
+                                    <title ref="LIT1113Anqasa" type="incomplete"/>
+                                    <note>The end is missing.</note>
+                                    <explicit xml:lang="gez">ሰሎምን፡ አቡኪ፡ ተነበየ፡ ወይቤ፡ ጼና፡ አልባስኪ፡ ከመ፡ ፄና፡ ስኂን፡ ገነት፡ ዕፁት፡ እኅትየ፡ መርዓት፡ ገነት፡ ዕፁት፡ ወዐዘቅት፡ 
+                                        ሕትምት፡ ፍናወ፡ ዚአኪ። ገነት፡ ምስለ፡ ፍሬ፡ አቅማ<supplied reason="lost" resp="PRS8999Strelcyn">ሕ፡</supplied></explicit>
+                                </msItem>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">195</measure>
+                                        <dimensions type="outer" unit="mm">
+                                            <height>220</height>
+                                            <width>180</width>
+                                        </dimensions>
+                                    </extent>
+                                    <collation>23 quires which, with a few exceptions, are numbered. The first has the number 2, indicating, that the first 
```

Comment:
maybe the element` <signatures>` could be used to give the information on quire marks?